tests/solidity/test/ComplianceBridge.js

```diff
@@ -1,36 +1,87 @@
-const { expect } = require("chai");
+const { expect } = require("chai")
 const { ethers } = require("hardhat")
 const { sendShieldedTransaction, sendShieldedQuery } = require("./testUtils")
 
+const unencryptedProvider = new ethers.providers.JsonRpcProvider("http://localhost:8547") // Unencrypted rpc url
+
 describe('ComplianceBridge', () => {
+    const CONTRACT_ADDRESS = "0x30252afe8c1683fd184c99a3c44aa5d547d59dd4"
     let contract
 
     before(async () => {
         const [signer] = await ethers.getSigners()
         const ComplianceProxyFactory = await ethers.getContractFactory('ComplianceProxy')
-        contract = new ethers.Contract("0x30252afe8c1683fd184c99a3c44aa5d547d59dd4", ComplianceProxyFactory.interface, signer);
+        contract = new ethers.Contract(CONTRACT_ADDRESS, ComplianceProxyFactory.interface, signer)
     })
 
     it('Should be able to add verification details', async () => {
         const [signer] = await ethers.getSigners()
 
+        // Make sure that signer was not verified yet
+        const isVerifiedRespBeforeCall = await sendShieldedQuery(
+            signer.provider,
+            contract.address,
+            contract.interface.encodeFunctionData("isUserVerified", [signer.address])
```

Comment:
Try to verify random eth address, not the same. It will make test able to run multiple times